ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -0,0 +1,1460 @@
+              description: in IP address format A.B.C.D
+              type: str
+            timers:
+              type: str
+              description: Time between BGP keepalive messages in seconds "<0-3600> <0-3600>"
+            networks:
+              type: list
+              primary_key: prefix
+              convert_types:
+                - dict
+              items:
+                type: dict
+                keys:
+                  prefix:
+                    type: str
+                    description: IPv4 prefix "A.B.C.D/E" or IPv6 prefix "A:B:C:D:E:F:G:H/I"
+                  route_map:
+                    type: str
+            listen_ranges:
+              type: list
+              description: |
+                New improved "listen_ranges" data model to support multiple listen ranges and additional filter capabilities
+              items:
+                type: dict
+                keys:
+                  prefix:
+                    type: str
+                    description: IPv4 prefix "A.B.C.D/E" or IPv6 prefix "A:B:C:D:E:F:G:H/I"
+                  peer_id_include_router_id:
+                    type: bool
+                    description: Include router ID as part of peer filter
+                  peer_group:
+                    type: str
+                    description: Peer-group name
+                  peer_filter:
+                    type: str
+                    description: |
+                      Peer-filter name
+                      note: `peer_filter`` or `remote_as` is required but mutually exclusive.
+                      If both are defined, peer_filter takes precedence
+                  remote_as:
+                    type: str
+                    description: BGP AS <1-4294967295> or AS number in asdot notation <1-65535>.<0-65535>
+                    convert_types:
+                      - int
+            neighbors:
+              type: list
+              primary_key: ip_address
+              convert_types:
+                - dict
+              items:
+                type: dict
+                keys:
+                  ip_address:
+                    type: str
+                  peer_group:
+                    type: str
+                    description: Peer-group name
+                  remote_as:
+                    type: str
+                    description: BGP AS <1-4294967295> or AS number in asdot notation <1-65535>.<0-65535>
+                    convert_types:
+                      - int
+                  password:
+                    type: str
+                  remove_private_as:
+                    type: dict
+                    description: Remove private AS numbers in outbound AS path
+                    keys:
+                      enabled:
+                        type: bool
+                      all:
+                        type: bool
+                      replace_as:
+                        type: bool
+                      ingress:
```

Comment:
This is not documented and not valid according to the template.

Reply:
Changed hostvars